internal/lsp/server.go

```diff
@@ -2004,11 +1998,7 @@ func (l *LanguageServer) handleTextDocumentFormatting(
 	switch formatter {
 	case "opa-fmt", "opa-fmt-rego-v1":
 		opts := format.Opts{
-			RegoVersion: rules.RegoVersionFromVersionsMap(
-				l.loadedConfigAllRegoVersions.Clone(),
-				strings.TrimPrefix(params.TextDocument.URI, l.workspaceRootURI),
-				ast.RegoUndefined,
-			),
+			RegoVersion: l.regoVersionForURI(params.TextDocument.URI),
```

Comment:
This is just DRYing up with some more use of regoVersionForURI and is otherwise unrelated.